src/main/java/it/gov/pagopa/merchant/controller/MerchantController.java

```diff
@@ -29,4 +33,7 @@ ResponseEntity<MerchantDetailDTO> getMerchantDetail(
             @PathVariable("initiativeId") String initiativeId,
             @PathVariable("merchantId") String merchantId
     );
+    @GetMapping("/initiatives")
```

Comment:
add a new line separator in order to make more readable this method.
Also, as already done on the above API, add @Operation annotation in order to describe te API